tsconfig.json

```diff
@@ -9,7 +9,10 @@
     "noImplicitReturns": false,
     "noUnusedLocals": true,
     "sourceMap": true,
-    "target": "esnext"
+    "target": "esnext",
+    // TODO The `skipLibCheck` setting can be removed when
+    // https://github.com/ReactiveX/rxjs/issues/3031 is resolved.
+    "skipLibCheck": true
```

Comment:
TS allows comments in the JSON file.

`skipLibCheck` just means typescript doesn't verify the declaration files of our deps. Usually it's good to have it enabled, there's just a small problem with the rxjs declarations that will hopefully get fixed soon.

Reply:
Wow, with TypeScript we can finally comment the `.json` files.